main.js

```diff
@@ -0,0 +1,38 @@
+
+
+function maxOfTwoNumbers(one, two){
+    Math.max(one, two);
+    console.log(Math.max(one, two))
+}
+maxOfTwoNumbers(3, 4);
+
+function maxOfThree(num1, num2, num3){
+    console.log(num1, num2, num3)
```

Comment:
you did great in the first function what's the problem? why you just log the numbers I want to get the max of three numbers